database/migrate_lexical_features_table.js

```diff
@@ -0,0 +1,107 @@
+export function migrate_lexical_features_table(project, targets_db) {
+	create_tabitha_table(targets_db)
+
+	load_data(targets_db, project)
+}
+
+/** @param {import('bun:sqlite').Database} targets_db */
+function create_tabitha_table(targets_db) {
+	create()
+
+	empty_existing_table()
+
+	return targets_db
+
+	function create() {
+		console.log(`Creating Lexical_Features table in ${targets_db.filename}...`)
+
+		targets_db.query(`
+			CREATE TABLE IF NOT EXISTS Lexical_Features (
+				project			TEXT,
+				part_of_speech	TEXT,
+				feature			TEXT,
+				position			INTEGER,
+				code				TEXT,
+				value				TEXT,
+				notes				TEXT
+			)
+		`).run()
+
+		console.log('done.')
+	}
+
+	function empty_existing_table() {
+		console.log(`Cleaning existing data in Lexical_Features...`)
+
+		targets_db.query(`DELETE FROM Lexical_Features`).run()
+
+		console.log('done.')
+	}
+}
+
+/** @param {import('bun:sqlite').Database} targets_db */
+function load_data(targets_db, project) {
+	console.log(`Loading data into Lexical_Features table...`)
+
+	const data = [
+		[project, 'Noun', 'Common/Proper', 1, 'C', 'Common', 'Articles only modify common nouns.  *the John'],
```

Comment:
These should actually be automatically extracted from the .mdb table, rather than hard-coded. There's no point passing 'project' in as a parameter if we hard-code the features, as these features are only in the English project.

Same goes for the Form_Names.

Reply:
✅ [c5446dc](https://github.com/presciencelabs/tabitha-targets/pull/7/commits/c5446dc444e5179268d851e22bc9088202c86106) for `Lexical_Features`

⌛ working on `Form_Names` next.

Reply:
ok, ✅ [ff73188](https://github.com/presciencelabs/tabitha-targets/pull/7/commits/ff7318817ec7809119b090b9c30ef449225ec6b5) for `Form_Names`